### PR TITLE
dsp: add host benchmark

### PR DESCRIPTION
**Pull request by jordens**

### CODE REVIEW

**Comment by jordens** [comment]
bors try

**Comment by jordens** [comment]
bors merge

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1 @@
 /target
-/dsp/target
```
Comment (matthuszagh) [inline .gitignore:2]: Was this change intentional?

Comment (jordens) [reply .gitignore:2]: Yes. Not needed with workspaces iirc.
